locale/nl_NL.UTF-8/LC_MESSAGES/palma.po

```diff
@@ -0,0 +1,514 @@
+# PalMA translation.
+# Copyright (C) 2017-2024 Universitätsbibliothek Mannheim
+# This file is distributed under the same license as the PalMA package.
+# Stefan Weil <stefan.weil@bib.uni-mannheim.de>, 2014.
+# Dennis Müller <dennis.mueller@bib.uni-mannheim.de>, 2017.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PalMA 0.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-06-29 18:50+0200\n"
+"PO-Revision-Date: 2017-09-12 14:38+0200\n"
+"Last-Translator: Dennis Müller <dennis.mueller@bib.uni-mannheim.de>\n"
+"Language-Team: English <en@li.org>\n"
```

Comment:
The basic structure looks fine. If you want to add Dutch translations, you should update the meta information like last translator or language team, too, of course. Without at least some translations, the newly added file won't help.

Reply:
Also note that all language contributions are listed in the [README file](https://github.com/UB-Mannheim/PalMA/blob/master/locale/README.md). This requires that your author name, which also appears in the "Signed-off-by" comment, is configured correctly. At the moment, as far as I can see, it isn't.

Reply:
@Max-0607, are you planning more work for this pull request? Or should it be closed because I cannot merge it in its current state (see my comments above)?